onnxruntime/hosting/server_configuration.h

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#ifndef ONNXRUNTIME_HOSTING_SERVER_CONFIGURATION_H
+#define ONNXRUNTIME_HOSTING_SERVER_CONFIGURATION_H
+
+#include <thread>
+#include <fstream>
+
+#include "boost/program_options.hpp"
+
+namespace onnxruntime {
+namespace hosting {
+
+namespace po = boost::program_options;
+
+// Wrapper around Boost program_options and should provide all the functionality for options parsing
+// Provides sane default values
+class ServerConfiguration {
+ public:
+  ServerConfiguration() {
+    desc.add_options()("help,h", "Shows a help message and exits");
+    desc.add_options()("model_path,m", po::value(&model_path)->required(), "Path to ONNX model");
+    desc.add_options()("address,a", po::value(&address)->default_value(address), "The base HTTP address");
+    desc.add_options()("port,p", po::value(&port)->default_value(port), "HTTP port to listen to requests");
```

Comment:
`port` -> `http_port`? We will have gRPC later and at that time we need different port.

Reply:
Agreed.